source/features/github-bugs.css

```diff
@@ -74,6 +74,13 @@ div.markdown-body {
 	min-height: 2em !important;
 }
 
+/* Sponsor button in mobile view #7853 */
```

Comment:
Thank you! Can you add a test URL? Just our repo

This does not affect the button on large windows?

Reply:
It works fine.

https://github.com/user-attachments/assets/5ee87312-3139-485b-8d5a-aa01c96825ac